docs/sftt/sftt-api-spec.md

```diff
@@ -1681,3 +1681,92 @@ If the site id specified is invalid.
 ##### `401 UNAUTHORIZED`
 
 If the calling user is not an admin.
+
+### Record Stewardship Activity
+
+`POST api/v1/protected/sites/:site_id/record_stewardship`
+
+Records a stewardship activity for the given site. Indicate `True` if the activity (watered, mulched, etc.) was completed, else `False`.
+
+#### Request Body
+
+```json
+{
+  "date": TIMESTAMP,
+  "duration": INT | NULL,
```

Comment:
Should we require users to enter a duration? It doesn't seem like a lot to ask and might be helpful information

Reply:
SFTT specifically asked we don't require it. If you want to require it it's an easy fix, we can ask tomorrow.